src/components/tile/OptionsTile.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Card } from 'react-bootstrap';
+
+import { EllipsisDropdown } from '@/components';
+
+import './Tile.scss';
+
+type Props = {
+  tileHeader: string;
+  children: React.ReactNode;
+  dropdownHeader: string;
+  dropdownItems: { label: string; value: string }[];
+  dropdownOnClick: (code: string) => void;
+};
```

Comment:
możesz tutaj wykorzystać `React.ComponentPropsWithoutRef<...>` czy jakoś tak i podać komponent, którego propsy chcesz przyjąć jako typ. Jeśli chcesz jakieś z nich ominąć to wrzucasz to w Omit<React.Component..., "to co chcesz ominąć">. Wówczas skróci to zapis i nie będziesz musiał ich destrukturyzować.  Wystarczy wtedy zapis: props i później przekażesz do komponentu podrzędnego {...props}

Reply:
👍